solution/ui/regulations/eregs-vite/src/utilities/utils.js

```diff
@@ -48,9 +49,11 @@ function parseError(err) {
         error.requestId = requestId;
         error.root = err;
         error.status = status;
-
         return error;
     } catch {
+        if(err.detail){
```

Comment:
we need this to use the error message we provide from django as a value for logic